api/service.py

```
@@ -634,6 +649,93 @@ async def get_anomaly_history(
+    return APIKeyCreateResponse(
+        id=api_key.id,
+        name=api_key.name,
+        key=api_key.key,
+        permissions=api_key.permissions,
+        created_at=api_key.created_at,
+        expires_at=api_key.expires_at
+    )
+
+
+@app.get("/api/v1/auth/apikeys", response_model=List[APIKeyResponse])
+async def list_api_keys(current_user: User = Depends(get_current_user)):
```

Ruff (0.14.10)

Undefined name `User`

(F821)